arches/app/views/manifest_manager.py

```diff
@@ -195,14 +195,21 @@ def change_canvas_label(manifest, canvas_id, label):
 
         files = request.FILES.getlist("files")
         name = request.POST.get("manifest_title")
+        if name == "null" or name == "undefined":
+            try:
+                name = os.path.splitext(files[0].name)[0]
+            except:
+                pass
+
         attribution = request.POST.get("manifest_attribution", "")
         logo = request.POST.get("manifest_logo", "")
-        desc = request.POST.get("manifest_description")
+        desc = request.POST.get("manifest_description", "")
         operation = request.POST.get("operation")
         manifest_url = request.POST.get("manifest")
         canvas_label = request.POST.get("canvas_label")
         canvas_id = request.POST.get("canvas_id")
         metadata = request.POST.get("metadata")
+        print(metadata)
```

Comment:
Remove

Reply:
Thanks.
The print statement has been removed.
Also, dirty state is restored by updating the part to display logo.